File: JavaScript/TowerOfHanoi.js
const tower = (n, rod_initial,  rod_final,  rod_temp) =>{
        if (n == 0)
        {
            return;
        }
        tower(n - 1, rod_initial, rod_temp, rod_final);
        console.log("disk " + n + " initial rod " + rod_initial +
        " final rod " + rod_final);
        tower(n - 1, rod_temp, rod_final, rod_initial);
}

const disks = 3;
     
tower(disks, 'Disk 1', 'Disk 2', 'Disk 3');

File: Python/TowerOfHanoi.py
def TowerOfHanoi(disks , initialRod, finalRod, tempRod):
    if disks==1:
        print ("disk 1 from initialRod",initialRod,"to finalRod.",finalRod)
        return
    TowerOfHanoi(disks-1, initialRod, tempRod, finalRod)
    print ("disk",disks,"from initialRod",initialRod,"to finalRod.",finalRod)
    TowerOfHanoi(disks-1, tempRod, finalRod, initialRod)

disks = 4
TowerOfHanoi(disks,'Disk 1','Disk 2','Disk 3')